packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -4,6 +4,22 @@ import { css } from '@patternfly/react-styles';
 import { DrawerColorVariant, DrawerContext } from './Drawer';
 import { formatBreakpointMods } from '../../helpers/util';
 import { GenerateId } from '../../helpers/GenerateId/GenerateId';
+import { FocusTrap } from '../../helpers/FocusTrap/FocusTrap';
+
+export interface DrawerPanelFocusTrapObject {
+  /** Enables a focus trap on the drawer panel content. This will also automatically
+   * handle focus management when the panel expands and when it collapses. Do not pass
+   * this prop if the isStatic prop on the drawer component is true.
+   */
+  enabled?: boolean;
+  /** The element to focus when the drawer panel content expands. By default the
+   * first focusable element will receive focus. If there are no focusable elements, the
+   * panel itself will receive focus.
+   */
+  elementToFocusOnExpand?: HTMLElement | SVGElement | string;
+  /** One or more id's to use for the drawer panel content's accessible label. */
+  'aria-labelledby'?: string;
```

Comment:
Why is this only needed when we have focus trap enabled?

Reply:
With the focus trap enabled we're adding a `role="dialog"` to the panel content. MDN has some context on [labelling a dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role#labeling). For the non-focus trap Drawer, we just have a plain `div` which typically doesn't do well with aria labelling.